tests/sourcetransform/test_messages.py

```diff
@@ -35,9 +35,9 @@ def test_message_to_drop(self):
             "Keys": [],
             "Value": b"",
             "Tags": [DROP],
-            "EventTime": EVENT_TIME_FOR_DROP,
+            "EventTime": mock_event_time(),
         }
-        msgt = Message(b"", mock_event_time()).to_drop()
+        msgt = Message(b"", datetime(1, 1, 1, 0, 0)).to_drop(mock_event_time())
```

Comment:
Can you add an assert to check if the specific event time passed matches the one contained in the object?

Reply:
just curious, why do you want that check? it is totally ok for it not to match.

Reply:
Then how do we know the code does what it does? :)

Reply:
+1 to Vigith, I actually intentionally set the original event time as different from the new one we assign to the dropped message. Such that the unit test verifies the update.

Reply:
Sounds good. You can merge then

Reply:
e.g., where it might not match is, say, we have a payload with event-time set to 10 years into the future. we can drop the message while setting the event-time to `time.Now`.